src/components/application_manager/test/include/application_manager/resumption_data_test.h

```diff
@@ -116,13 +123,13 @@ class ResumptionDataTest : public ::testing::Test {
   void CheckVRTitle(sm::SmartObject& res_list);
   void CheckSubscriptions(sm::SmartObject& res_list);
 
-  const size_t count_of_commands = 5;
-  const size_t count_of_choice = 2;
-  const size_t count_of_choice_sets = 4;
-  const size_t count_of_submenues = 3;
-  const size_t count_of_files = 8;
-  const size_t count_of_vrhelptitle = 2;
-  const std::string mac_address_ = "12345";
+  const size_t kCountOfCommands_;
```

Comment:
@VVeremjova I'm not sure about this naming style correctness. I can't find such instructions in Google coding style